apps/github/src/@types/index.d.ts

```diff
@@ -0,0 +1,30 @@
+type Payload = {
+    action: string;
+    sender: { [key: string]: unknown };
+    repository: { [key: string]: unknown };
+    organization: { [key: string]: unknown };
+    installation: { [key: string]: unknown };
+    [key: string]: unknown;
+};
+
+type Issue = {
+    body: string | null;
+    title: string;
```

Comment:
If we just need `body` and `title` it's okay to define only them although the payload has other props.

Reply:
![image](https://user-images.githubusercontent.com/44132059/213743435-485ab505-c1b2-4c6f-8268-5520cfe69806.png)
I got this error when I removed the extra props.